terraform/projects/app-ecs-services/prometheus-service.tf

```diff
@@ -66,6 +66,14 @@ data "aws_iam_policy_document" "prometheus_policy_doc" {
   }
 }
 
+resource "aws_route53_record" "prometheus_dns" {
```

Comment:
Can this be moved to the infra-networking-route53 project please.

Reply:
See comment above

Reply:
@kentsanggds I don't think `infra-networking-route53` should own all route53 records. That's not a good way of structuring our projects.

Reply:
I have moved this to the route53 folder. I will wait until we move it to networking, then I will add the records back into the services section.